database/console/show_command_test.go

```diff
@@ -0,0 +1,135 @@
+	mocksconfig "github.com/goravel/framework/mocks/config"
+	mocksconsole "github.com/goravel/framework/mocks/console"
+	mocksorm "github.com/goravel/framework/mocks/database/orm"
+	mocksschema "github.com/goravel/framework/mocks/database/schema"
+	"github.com/goravel/framework/support/color"
+)
+
+func TestShowCommand(t *testing.T) {
+	var (
+		mockContext *mocksconsole.Context
+		mockConfig  *mocksconfig.Config
+		mockSchema  *mocksschema.Schema
+		mockOrm     *mocksorm.Orm
+		mockQuery   *mocksorm.Query
+	)
+
+	beforeEach := func() {
+		mockContext = mocksconsole.NewContext(t)
+		mockConfig = mocksconfig.NewConfig(t)
+		mockSchema = mocksschema.NewSchema(t)
+		mockOrm = mocksorm.NewOrm(t)
+		mockQuery = mocksorm.NewQuery(t)
+	}
+
+	tests := []struct {
+		name     string
+		setup    func()
+		expected string
+	}{
+		{
+			name: "invalid argument",
+			setup: func() {
+				mockContext.EXPECT().Argument(0).Return("test")
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
+					color.Errorln(message)
+				})
+			},
+			expected: "No arguments expected for 'db:show' command, got 'test'.",
+		},
+		{
+			name: "get tables failed",
+			setup: func() {
+				mockContext.EXPECT().Argument(0).Return("")
+				mockContext.EXPECT().Option("database").Return("")
+				mockSchema.EXPECT().Connection(mock.Anything).Return(mockSchema)
+				mockSchema.EXPECT().GetConnection().Return("test")
+				mockConfig.EXPECT().GetString(mock.Anything).Return("test")
+				mockQuery.EXPECT().Driver().Return(database.DriverMysql)
+				mockOrm.EXPECT().Query().Return(mockQuery)
+				mockSchema.EXPECT().Orm().Return(mockOrm)
+				mockQuery.EXPECT().Raw(mock.Anything).Return(mockQuery)
+				mockQuery.EXPECT().Scan(mock.Anything).Return(nil)
+				mockSchema.EXPECT().GetTables().Return(nil, assert.AnError)
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
+					color.Errorln(message)
+				})
+			},
+			expected: assert.AnError.Error(),
+		}, {
+			name: "get views failed",
+			setup: func() {
+				mockContext.EXPECT().Argument(0).Return("")
+				mockContext.EXPECT().Option("database").Return("")
+				mockSchema.EXPECT().Connection(mock.Anything).Return(mockSchema)
+				mockSchema.EXPECT().GetConnection().Return("test")
+				mockConfig.EXPECT().GetString(mock.Anything).Return("test")
+				mockQuery.EXPECT().Driver().Return(database.DriverMysql)
+				mockOrm.EXPECT().Query().Return(mockQuery)
+				mockSchema.EXPECT().Orm().Return(mockOrm)
+				mockQuery.EXPECT().Raw(mock.Anything).Return(mockQuery)
+				mockQuery.EXPECT().Scan(mock.Anything).Return(nil)
+				mockSchema.EXPECT().GetTables().Return(nil, nil)
+				mockContext.EXPECT().OptionBool("views").Return(true)
+				mockSchema.EXPECT().GetViews().Return(nil, assert.AnError)
+				mockContext.EXPECT().Error(mock.Anything).Run(func(message string) {
+					color.Errorln(message)
+				})
+			},
+			expected: assert.AnError.Error(),
+		}, {
+			name: "success",
```

Comment:
No expected in this case? We can use `Contains` to assert it if it's a bit complicated.